### PR TITLE
Update the relative file paths in the comments of tf.data kernel files

### LINKED ISSUE #22576: Confusing information in the tf.data Op kernel files
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: no
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: MacOS
- **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device**:
- **TensorFlow installed from (source or binary)**: source
- **TensorFlow version (use command below)**: 1.11.0
- **Python version**: 3.6.5
- **Bazel version (if compiling from source)**: 0.16.0
- **GCC/Compiler version (if compiling from source)**: 4.2.1
- **CUDA/cuDNN version**: N/A
- **GPU model and memory**: N/A
- **Exact command to reproduce**: N/A

### Describe the problem
A bunch of files for tf.data Ops under the directory `tensorflow/core/kernels/data/` has the following doc:

```
// See documentation in ../ops/dataset_ops.cc for a high-level
// description of the following op.
```
However, I could not find this file at the referred location (`../ops/dataset_ops.cc`). Does it mean the file `tensorflow/core/ops/dataset_ops.cc`? If yes, I would like to submit a PR to update them.

### PR DESCRIPTION
**Pull request by feihugis**
`tf.data` kernel files are moved down one directory in [this commit](https://github.com/tensorflow/tensorflow/commit/a5b2a0c9a3335d10c4dd3dfdff96149f74a4d120#diff-1362ec26d9480143607b5c5609959280). Accordingly, this PR updates the relative paths in several comments of `tf.data` kernel files that need to be changed.

Fixes #22576

### CODE REVIEW

**Review by mrry — APPROVED** [review]

**Comment by tensorflowbutler** [comment]
Nagging Assignee @caisq: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by feihugis** [comment]
@caisq This PR has been a while since the approval. Is there anything I need to do?

**Comment by caisq** [comment]
@feihugis The change is being tested and reviewed internally. When it is submitted internally, this GH PR will be automatically merged. No actions is required from you at this time. Thank you for your patience.

**Comment by caisq** [comment]
And sorry for the delay.

**Comment by feihugis** [comment]
@caisq Thanks for your quick reply. The tests for `MacOS Python2 and CC` failed, but I think it may be not related as this PR only changes the documentation. Do you know why `feedback/copybara` failed?

**Comment by drpngx** [comment]
@feihugis internal tests are running... crossing fingers.

**Comment by drpngx** [comment]
Passes. Waiting for internal approval.

**Pull request merged**
